pkg/api/v1/client/testworkflow.go

```diff
@@ -0,0 +1,80 @@
+package client
```

Comment:
It's just an API client, that is using public endpoints - the endpoints are TCL-licensed, and have Pro checks. Because of that, similarly to the OpenAPI schema/models, I don't see a reason to keep that TCL licensed